internal/tui/model_list.go

```diff
@@ -36,16 +52,31 @@ type ListModel struct {
 	description string
 	suggestions []string
 	list        list.Model
+	input       textinput.Model
+	keys        keyMap
+	help        help.Model
 
 	confirmed bool
 	loading   bool
 
-	spinner  spinner.Model
-	selected string
+	spinner   spinner.Model
+	selected  item
+	usePrefix bool
 
 	err error
 }
 
+var listKeys = []key.Binding{
+	key.NewBinding(
+		key.WithKeys("t"),
+		key.WithHelp("t", "toggle suffix"),
```

Comment:
this should be `toggle prefix` right? suffix is ending, prefix is preceding?

Reply:
Yes that's right my bad :/